### PR TITLE
added:

### PR DESCRIPTION
**Pull request by IlyaSibogatov**
buttons;
colors;
pictures;
info section

### CODE REVIEW

File: app/src/main/java/com/example/lessson17/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,116 @@
...
+
 
 class MainActivity : AppCompatActivity() {
+    @SuppressLint("SimpleDateFormat", "SetTextI18n")
     override fun onCreate(savedInstanceState: Bundle?) {
+
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        var COUNT = 0
```
Comment (vshat-tms) [inline app/src/main/java/com/example/lessson17/MainActivity.kt]: var + CAPS??

File: app/src/main/java/com/example/lessson17/MainActivity.kt [hunk 2]
```diff
@@ -1,11 +1,116 @@
...
+
+        //findViewById
+        val INFO_TABLE = findViewById<TextView>(R.id.info)
```
Comment (vshat-tms) [inline app/src/main/java/com/example/lessson17/MainActivity.kt]: don't use caps

File: app/src/main/java/com/example/lessson17/MainActivity.kt [hunk 3]
```diff
@@ -1,11 +1,116 @@
...
+        val BACKGROUND = findViewById<LinearLayout>(R.id.background)
+        val IMAGE: ImageView = findViewById(R.id.image)
+
+        //PHONE DATA
```
Comment (vshat-tms) [inline app/src/main/java/com/example/lessson17/MainActivity.kt]: move consts to companion

File: app/src/main/java/com/example/lessson17/MainActivity.kt [hunk 4]
```diff
@@ -1,11 +1,116 @@
...
+        val PHONE_MANUFACTURE = Build.MANUFACTURER
+        val PHONE_MODEL = Build.MODEL
+        val DATE_FORMAT = SimpleDateFormat("HH:mm:ss")
+
+        //TOAST
+        val toastText = "Тут могла быть ваша реклама"
+        val duration = Toast.LENGTH_LONG
+        val toast = Toast.makeText(applicationContext, toastText, duration)
```
Comment (vshat-tms) [inline app/src/main/java/com/example/lessson17/MainActivity.kt]: create toast on demand

File: app/src/main/java/com/example/lessson17/MainActivity.kt [hunk 5]
```diff
@@ -1,11 +1,116 @@
...
+        }
+        findViewById<Button>(R.id.time).setOnClickListener {
+            val currentDate = DATE_FORMAT.format(Date())
+            INFO_TABLE.text = currentDate
+        }
+        findViewById<Button>(R.id.toast).setOnClickListener {
+            toast.show()
+        }
+        findViewById<Button>(R.id.colorRed).setOnClickListener {
+            INFO_TABLE.setTextColor(Color.parseColor("red"))
```
Comment (vshat-tms) [inline app/src/main/java/com/example/lessson17/MainActivity.kt]: use `Color.RED`

File: app/src/main/java/com/example/lessson17/MainActivity.kt [hunk 6]
```diff
@@ -1,11 +1,112 @@
...
+                2 -> image.setImageResource(R.drawable.dog)
+                3 -> image.setImageResource(R.drawable.frog)
+            }
+        }
+    }
+
+    companion object {
+        val PHONE_MANUFACTURE = Build.MANUFACTURER
+        val PHONE_MODEL = Build.MODEL
+        private val DATE_FORMAT = SimpleDateFormat("HH:mm:ss")
```
Comment (PetrGrebnev) [inline app/src/main/java/com/example/lessson17/MainActivity.kt:110]: Approve

**Review by PetrGrebnev — APPROVED** [review]

File: app/src/main/java/com/example/lessson17/MainActivity.kt (hunk 1 above)
Comment (IlyaSibogatov) [reply app/src/main/java/com/example/lessson17/MainActivity.kt]: CAPS changed to lowercase

File: app/src/main/java/com/example/lessson17/MainActivity.kt (hunk 2 above)
Comment (IlyaSibogatov) [reply app/src/main/java/com/example/lessson17/MainActivity.kt]: changed

File: app/src/main/java/com/example/lessson17/MainActivity.kt (hunk 3 above)
Comment (IlyaSibogatov) [reply app/src/main/java/com/example/lessson17/MainActivity.kt]: added companion object

File: app/src/main/java/com/example/lessson17/MainActivity.kt (hunk 4 above)
Comment (IlyaSibogatov) [reply app/src/main/java/com/example/lessson17/MainActivity.kt]: toast changed

File: app/src/main/java/com/example/lessson17/MainActivity.kt (hunk 5 above)
Comment (IlyaSibogatov) [reply app/src/main/java/com/example/lessson17/MainActivity.kt]: color substitution methods changed

File: app/src/main/java/com/example/lessson17/MainActivity.kt (hunk 6 above)
Comment (IlyaSibogatov) [reply app/src/main/java/com/example/lessson17/MainActivity.kt:110]: tnx

**Review by vshat-tms — APPROVED** [review]

**Pull request closed without merging**